DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeatures.c

```diff
@@ -646,6 +647,18 @@ DasharoSystemFeaturesCallback (
        */
       Value->u8 = FixedPcdGet8(PcdCpuMaxTemperature) - Private->DasharoFeaturesData.CpuThrottlingOffset;
       break;
+    case IBG_SACM_INFO_MSR_QUESTION_ID:
+      Value->u64 = AsmReadMsr64(0x13A);

```

Comment:
Reading an inexistent MSR may cause trouble (i.e. exception) on some platforms. I think GLK and SNB/IVB does not have this MSR. Thus the `disableif` suggested in other comment, so that this will never get called on platforms that are not supposed to support it (but still may have ME disable/enable option).